doc/deployment/google_cloud_platform.md

```diff
@@ -160,8 +160,28 @@ $ pachctl port-forward &
 And you're done! You can test to make sure the cluster is working by trying `pachctl version` or even creating a new repo.
 
 ```sh
+
 $ pachctl version
 COMPONENT           VERSION
 pachctl             1.8.0
 pachd               1.8.0
 ```
+
+### Additional Tips for Performance Improvements
+#### Increasing Ingress Throughput
+
+One way to improve Ingress performance is to restrict Pachd to a specific, more powerful node in the clustomer. This is accomplished by the use of [node-taints](https://cloud.google.com/kubernetes-engine/docs/how-to/node-taints) in GKE. By creating a node-taint for Pachd, we’re telling the kubernetes scheduler that the only workload that should be on that node is Pachd. Once that’s completed, you then deploy Pachyderm with the `--pachd-cpu-request` and `--pachd-memory-request` set to the match the resources limits of the machine type. And finally, you’ll modify the Pachd deployment such that it has an appropriate toleration:
```

Comment:
Clustomer lol?

Reply:
I would change "only workload that should be on that node" to "only pod that should be on that node"